backend/src/test/java/codezap/global/ServiceTest.java

```diff
@@ -41,4 +47,16 @@ public class ServiceTest {
 
     @Autowired
     protected LikesRepository likesRepository;
+
+    protected Template createSavedTemplate() {
+        Member member = memberRepository.save(MemberFixture.getFirstMember());
+        Category category = categoryRepository.save(CategoryFixture.getDefaultCategory(member));
+        return templateRepository.save(TemplateFixture.get(member, category));
+    }
+
+    protected Template createSavedSecondTemplate() {
```

Comment:
```suggestion
    protected Template createAnotherMemberTemplate() {
```
```suggestion
    protected Template createMember2Template() {
```
메서드 이름 만으론 "다른 멤버의 템플릿"을 만드는구나 라는 것을 알아보기가 힘들어요.
조금 길어지더라도 의미가 명확해야 하는 메서드라고 느껴져서 네이밍 변경 요청 드려요!